cpp/dolfinx/mesh/utils.h

```diff
@@ -108,5 +108,24 @@ Eigen::Array<std::int32_t, Eigen::Dynamic, 1> locate_entities_boundary(
         const Eigen::Ref<const Eigen::Array<double, 3, Eigen::Dynamic,
                                             Eigen::RowMajor>>&)>& marker);
 
+/// Compute the geometry indices of given entities from the mesh geometry
+/// @param[in] mesh Mesh
+/// @param[in] dim Topological dimension of the entities of interest
+/// @param[in] entity_list List of entities to obtain vertex geometry
+/// @param[in] orient If true, in 3D, reorients facets to have consistent normal
+/// direction
+/// @return Vertex indices in the geometry dofmap of the mesh for each entity
```

Comment:
`/// @return Indices in the geometry array for the mesh entity vertices, e.g.  indices(i, j) is the position in the geometry array of the j-th vertex of the entity entity_list[i].`